docs/synthetics/http-monitor.md

```diff
@@ -0,0 +1,68 @@
+title: HTTP Monitor
+description: HTTP monitor can monitor any HTTP endpoint like API, Web URL, etc. 
+
+The HTTP monitor sends a single HTTP request to the specified URL with configured request settings. It records the response data like headers, body, error if any and the various timings.
+
+### Configuration
+
+#### General
+
+* **Name** - Name of the monitor. Max Length is 255 characters
+* **HTTP Method** - HTTP Method
+* **URL** - URL of the HTTP monitor of format `<http(s)>://hostname:port/path`
+* **Frequency** - Frequency to execute the monitor
+* **Locations** - List of locations to run the monitor
+
+#### Request Settings
+
+* **Headers** - List of HTTP headers to be sent
+* **Query Params** - List of request query parameters to be appended to the URL
+* **Cookies** - List of Cookies to be sent
+* **Body** - Request body data. Max length is 4096 characters
+
+Optionally, the header, query param, and cookie values can be marked a hidden, which means it will be hidden for the users. Currently, we do not encrypt the hidden values while storing it in the DB.
+
+Max length of the header, query param, and cookie name and value are 512 and 1024 characters respectively. A maximum of 100 rows for each of the headers, query params, and cookies is allowed.
+
+### Conditions
+
+Conditions can be configured on the response fields and metrics. The conditions are evaluated for every run result. All the conditions should pass for a run to be declared a passed. If any condition fails, the run will fail and the monitor will be marked as failing. Condition types supported are:
```

Comment:
All the conditions should pass for a run to be declared a passed. => All conditions must pass for a run to be declared as passed.